### PR TITLE
Fix an issue with concating empty frames

### PR DESCRIPTION
**Pull request by sklam**
The issue is revealed in dask_gdf.merge.

### CODE REVIEW

File: pygdf/column.py [hunk 1]
```diff
@@ -56,8 +57,9 @@ def _concat(cls, objs):
             mask = Buffer.from_empty(mem)
             null_count = 0
             for o in objs:
-                mask.extend(o._get_mask_as_column().to_gpu_array())
-                null_count += o._null_count
+                if len(o) > 0:
```
Comment (kkraus14) [inline pygdf/column.py]: Since we previously filter out any objects that are empty, do we still need this check here?

**Comment by sklam** [comment]
This is weird.  I think I pushed a commit during the last GH outage and now this PR doesn't see my update.

**Comment by sklam** [comment]
Ok, a force push sync it.

**Comment by kkraus14** [comment]
@sklam looks good to me. Merging, thanks!

**Pull request merged**